### DISCUSSION TITLE
unable to connect gnodeb and ue

### DISCUSSION THREAD

**Discussion by patelsri** [srsran/srsRAN_Project#388] [General Help] [answered]
Hello everyone,

I hope this message finds you well. I am currently working on the srsRAN gNB with srsUE project. While attempting to run the UE, I've encountered an issue where the RRC gets connected and released immediately.

Upon further investigation, it appears that the error might be related to the open5gs core. I've noticed HTTP response errors in both the general logs and the AMF log. Please feel free to correct me if my analysis is inaccurate.

![image](https://github.com/srsran/srsRAN_Project/assets/155114420/f1d26a52-46f1-460f-b87d-b81000fcb19d)

below is sceenshot of Subscriber Configuration
![image](https://github.com/srsran/srsRAN_Project/assets/155114420/6bd8defd-85c5-462f-acd0-a87089fc1ea0)
![image](https://github.com/srsran/srsRAN_Project/assets/155114420/c8269439-a6e7-42b2-a97e-49f2ef076226)
![image](https://github.com/srsran/srsRAN_Project/assets/155114420/d5af68fd-fa4b-449e-9149-9cc9606493ed)
![image](https://github.com/srsran/srsRAN_Project/assets/155114420/b6376829-7d9e-4e41-9539-e3a7c0f2bcf0)
![image](https://github.com/srsran/srsRAN_Project/assets/155114420/9040cca6-0471-4452-9a9c-36118a3869a6)

 can anyone please help me i have attached configuration files below can please check it 

[gnb_files.zip](https://github.com/srsran/srsRAN_Project/files/13789885/gnb_files.zip)

**Comment by ninjab3s** [comment]
It seems Open5GS cannot find the IMSI in the MongoDB database. In the Open5gs logs it looks like the AMF had problems starting. Please make sure all Open5gs components are started and running properly. If the problem persists can you provide a clean log where all components are started properly? Also, how do you start Open5GS?

**Comment by ffxxcad** [comment, marked as answer]
It looks like you are using V2.7.X of Open5gs. I encountered similar issues. Updating the mcc and mnc in the nrf.yaml fixed it for me.

**Reply by rajugbv** [reply]
I was also facing the same problem, but I was able to resolve it by changing the MNC and MCC in 'nrf.yaml'. May I ask how you found the solution? Did you decode it using logs?

**Reply by ffxxcad** [reply]
There was an answer [here](https://github.com/open5gs/open5gs/discussions/2782#discussioncomment-8001181)
